data/json/items/gun/shot.json

```diff
@@ -524,6 +524,41 @@
     "extend": { "flags": [ "RELOAD_ONE" ] },
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "shot": 2 } } ]
   },
+  {
+    "id": "shotgun_triple",
+    "copy-from": "shotgun_s",
+    "type": "GUN",
+    "name": { "str": "triple barrel shotgun" },
+    "description": "Pretty modern-looking shotgun, that has three barrels.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "triple_honcho",
+        "name": { "str": "Triple Honcho" },
+        "description": "Made by Charles Daly, this modern three-barrel shotgun has a unique balance on how much it cost, and how little firepower it can release with it's 18.5 inch barrels.  Luckily you do not need to pay the full price for it anymore."
```

Comment:
```suggestion
        "description": "Made by Charles Daly, this modern three-barrel shotgun had a unique balance between how much it cost and how little firepower it could release with its 18.5-inch barrels.  Luckily, you do not need to pay the full price for it anymore."
```